src/datasources/jwt/configuration/jwt.configuration.ts

```diff
@@ -1,5 +1,8 @@
 import { registerAs } from '@nestjs/config';
 
+/**
+ * @deprecated
+ */
```

Comment:
The idea was to keep it so that we can decide whether we want to remove all the localized configuration values or not. But as you mentioned, let's delete it for now as we can always revert the changes.
Done https://github.com/safe-global/safe-client-gateway/pull/2067/commits/196db4fd3bc702923e7102e99ad69fef810087f7